airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/metadata.py

```diff
@@ -130,7 +130,7 @@ def metadata_definitions(context: OpExecutionContext) -> List[LatestMetadataEntr
         metadata_entry = LatestMetadataEntry(
             metadata_definition=metadata_def,
             icon_url=icon_url,
-            last_modified=blob.last_modified,
+            last_modified=blob.updated.isoformat(),
```

Comment:
Curious, non-blocking: is this for formatting purposes mostly here?

Reply:
the `LastMetadataEntry.last_modified` attribute takes a string - I did not bother changing it to a datetime.